follow_fork/follow_fork.c

```diff
@@ -0,0 +1,123 @@
+#define _POSIX_SOURCE
+#define _GNU_SOURCE
+
+#include "undolr.h"
+#include "undolr_deprecated.h"
+
+#include <errno.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <string.h>
+#include <stdbool.h>
+#include <time.h>
+#include <stdlib.h>
+#include <dlfcn.h>
+#include <linux/limits.h>
+
+extern char *__progname;
+
+void attach_lr (int, char**, char**) __attribute__ ((constructor));
+
+static bool
+is_valid_process(pid_t pid)
```

Comment:
`is_undo_process` (and then return the opposite value than what you currently do)?
It's not clear to me without reading the code what “valid” means.

Reply:
renamed to `process_needs_recording` as it will make it clear what this function will determine